### ISSUE TITLE
Update to Angular 13

### ISSUE THREAD

**Issue by stephanrauh** [dj-fiorex/angular2-smart-table#25]
I've prepared a pull request updating the library to Angular 13.

**Comment by hexdecimal16** [comment]
please merge it as soon as possible

**Comment by stephanrauh** [comment]
Until they do, you can add this version to your package.json:

```
    "@stephanrauh/angular2-smart-table": "^2.0.9",
```

To make it work, you also have to create a file called `.npmrc` in your root directory (i.e. the directory of the `package.json`) and put this line into that file:
```
@stephanrauh:registry=https://npm.pkg.github.com
```

Just be aware that this is my temporary fork. I don't intend to maintain it forever.

**Comment by tim6400** [comment]
> Until they do, you can add this version to your package.json:
> 
> ```
>     "@stephanrauh/angular2-smart-table": "^2.0.9",
> ```
> 
> To make it work, you also have to create a file called `.npmrc` in your root directory (i.e. the directory of the `package.json`) and put this line into that file:
> 
> ```
> @stephanrauh:registry=https://npm.pkg.github.com
> ```
> 
> Just be aware that this is my temporary fork. I don't intend to maintain it forever.

I did what you suggest and it ask for a password:

```
npm ERR! code E401
npm ERR! Incorrect or missing password.
npm ERR! If you were trying to login, change your password, create an
npm ERR! authentication token or enable two-factor authentication then
npm ERR! that means you likely typed your password in incorrectly.
npm ERR! Please try again, or recover your password at:
npm ERR!     https://www.npmjs.com/forgot
```

**Comment by stephanrauh** [comment]
Yeah - I noticed this yesterday, too. I'm not sure if that's the way it should be, or if I did something wrong. It seems you need a GitHub account and a GitHub token. 

Follow the instructions at https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-to-github-packages. Stop at the "publishing a package" section. All you need is a GitHub token allowing you to read an npm package from GitHub and to log in to npm using this GitHub token.

BTW, if someone knows how to make GitHub packages public to npm, please tell me. I can't believe it's really that complicated.

**Comment by tim6400** [comment]
> Yeah - I noticed this yesterday, too. I'm not sure if that's the way it should be, or if I did something wrong. It seems you need a GitHub account and a GitHub token.
> 
> Follow the instructions at https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-to-github-packages. Stop at the "publishing a package" section. All you need is a GitHub token allowing you to read an npm package from GitHub and to log in to npm using this GitHub token.
> 
> BTW, if someone knows how to make GitHub packages public to npm, please tell me. I can't believe it's really that complicated.

Thank you, I might check that but i feel it's to complicated solution even if it's a temporary one.
It's been over a month, Why dont the oner merge it already?
Maybe it's time to deatch this library anyway..

Again, thank you for your time

**Comment by jupmorenor** [comment]
This fork also seems to be unmaintained :(

**Comment by uap-universe** [comment]
> This fork also seems to be unmaintained :(

@jupmorenor Stay tuned, Angular 13 support will arrive very soon.

**Comment by JonasPetry** [comment]
@uap-universe Any updates on this? Thanks in advance 🚀

**Comment by JonasPetry** [comment]
For everyone looking for Angular 13 Support: We forked this repo and published a working version. Feel free to use it until this repo is updated ✌️

https://www.npmjs.com/package/@cirqusde/angular2-smart-table

**Comment by uap-universe** [comment]
The changes are now merged in the recent development version. They will be published as an official npm package when the 2.1.0 milestone is complete. Until then, feel free to continue with Jonas fork or a local build (run `npm run build:lib`, copy the contents to your project and use a local path in your package.json instead of a version number).
